src/hoc/Aux/Aux.js

```diff
@@ -0,0 +1,3 @@
+const aux = (props) => props.children;
+
+export default aux;
```

Comment:
Not very clear of the purpose of this component.

Consider removing all together and handle auth state logic in app.js as per comment for layout/layout.js

Reply:
done